librz/analysis/p/analysis_avr.c

```diff
@@ -599,11 +597,10 @@ INST_HANDLER(dec) { // DEC Rd
 }
 
 INST_HANDLER(des) { // DES k
-	if (desctx.round < 16) { //DES
-		op->type = RZ_ANALYSIS_OP_TYPE_CRYPTO;
-		op->cycles = 1; //redo this
-		rz_strbuf_setf(&op->esil, "%d,des", desctx.round);
-	}
+	op->type = RZ_ANALYSIS_OP_TYPE_CRYPTO;
+	op->cycles = 1; //redo this
```

Comment:
Add `TODO` or `FIXME` please, it will make easier to see/find.

Reply:
Actually I don't know why this comment was there in the first place. `des` takes 1 cycle, or 2 cycles if following a non-des op, and since we can only insert a single value here, 1 would be ok.
I'll just remove the comment once travis is through.